index.html

```diff
@@ -438,20 +417,18 @@ <h1>Terminology</h1>
             <dt>
                 <dfn>Hypermedia Control</dfn>
             </dt>
-            <dd>Hypermedia controls are Web links [[!RFC8288]] and
-                request templates where the latter can be seen as forms
-                to fill in and to send to servers.</dd>
+            <dd>A serialization of a Protocol Binding in hypermedia, that is,
+                either a Web link [[!RFC8288]] for navigation or a Web form for
+                performing other operations. Forms can be seen as request templates
+                provided by the server to be completeted and sent by the client.</dd>
             <dt>
                 <dfn>Interaction Affordance</dfn>
             </dt>
             <dd>
-                Metadata of a Thing exposing <a>hypermedia controls</a>
-                that describe the possible protocol-level choices to
-                clients, thereby suggesting how clients may interact
-                with the Thing. Examples use of affordances includes,
-                but are not limited, getting and setting <a>properties</a>,
-                invoking <a>actions</a>, and subscribing to <a>events</a>.
-            </dd>
+                Metadata of a Thing that shows the possible choices to clients,
+                thereby suggesting how clients may interact with the Thing.
+                W3C WoT defines three types of Interaction Affordances:
+                Properties, Actions, and Events.</dd>
```

Comment:
call on 5.4.: there could be more than these interactions, so it should not be limited.
Add an extra sentence: 
_There are many types of potential affordances, but_ W3C WoT defines three types of Interaction Affordances ...